### PR TITLE
Fixes ChaosEngine example

### PR DESCRIPTION
**Pull request by aswathkk**
We need to specify the `appkind` in the chaosengine.yml

cc: @shubhamchaudhary, @rajdas98, @prateekpandey14

### CODE REVIEW

File: website/versioned_docs/version-0.6.0/example.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ spec:
     appns: litmus # App namespace
     # FYI, To see app label, apply kubectl get pods --show-labels
     applabel: "run=myserver" # App Label
+    appkind: deployment
```
Comment (chandankumar4) [inline website/versioned_docs/version-0.6.0/example.md]: This file change is not required! The `appKind` will comes in next release

Comment (aswathkk) [reply website/versioned_docs/version-0.6.0/example.md]: Does it comes in version 0.7.0?

Comment (chandankumar4) [reply website/versioned_docs/version-0.6.0/example.md]: No, it will comes in 0.8.0

**Review by chandankumar4 — APPROVED** [review]
Thanks @aswathkk for your contribution, Made some changes as per required.

**Comment by NisantaSahoo** [comment]
Thanks for taking part in Hacktoberfest 2019. Please fill up this form to claim your swag https://hubs.ly/H0l5ldm0

**Pull request merged**